src/main/java/ch/enterag/utils/mime/MimeTypes.java

```diff
@@ -1,788 +1,20 @@
 package ch.enterag.utils.mime;
 
-import java.util.*;
+import org.apache.tika.mime.MimeTypeException;
+
+import static org.apache.tika.mime.MimeTypes.getDefaultMimeTypes;
+
+public class MimeTypes {
+
+    public static String getExtension(String mimeType) throws MimeTypeException {
+        String extension = getDefaultMimeTypes().forName(mimeType).getExtension();
+        if (extension.isEmpty()) {
+            extension = "bin";
```

Comment:
It would be nicer to use a constant instead of the "magic string" bin here. Even if it is used only once.